tests/bugfixes/github/test_issue_981.py

```diff
@@ -48,19 +48,17 @@ class CanonAfInfoTest(metaclass=CaseMeta):
 Exif.Canon.AFPointsInFocus                   Short       4  25
 Exif.Canon.AFPointsSelected                  Short       4  25
 Exif.Canon.AFPointsUnusable                  Short       4  (none)
-Exif.Canon.AFMicroAdj                        Long       11  44 2 0 10 4294967295 0 10 0 10 0 10
-Exif.Canon.AFConfig                          Long       20  80 4 0 0 1 0 0 0 0 1 0 1 32639 1 0 1 0 0 0 2
 ""","""0x2602 Canon        AFNumPoints                 SShort      1  63
 0x2603 Canon        AFValidPoints               SShort      1  61
 0x260c Canon        AFPointsInFocus             Short       4  0 560 57344 0
 0x260d Canon        AFPointsSelected            Short       4  0 1848 57344 0
 0x260e Canon        AFPointsUnusable            Short       4  0 0 0 0
 ""","""Exif.Canon.AFNumPoints                       SShort      1  63
 Exif.Canon.AFValidPoints                     SShort      1  61
-Exif.Canon.AFPointsInFocus                   Short       4  20,21,25,45,46,47
-Exif.Canon.AFPointsSelected                  Short       4  19,20,21,24,25,26,45,46,47
-Exif.Canon.AFPointsUnusable                  Short       4  (none)
-""","""0x260e Canon        AFPointsUnusable            Short       4  3608 49152 792 6272
+Exif.Canon.AFPointsInFocus                   Short       4  (none)
+Exif.Canon.AFPointsSelected                  Short       4  25
+Exif.Canon.AFPointsUnusable                  Short       4  3,4,9,10,11,30,31,35,36,40,41,55,59,60
+""","""0x260e Canon        AFPointsUnusable            Short       4  0 0 0 0
```

Comment:
Did anyone check if this change makes sense? (e.g. validate against exiftool)

Reply:
@hassec Are you talking about every change relating to Canon tags, or specifically the AutoFocus changes?   I can't say anything about all the changes relating to Canon tags.  The AutoFocus tags were carefully researched and added to Exiv2 in August 2019 with a lot of input from @derselbst.  The code in this PR passes the tests which we added in 2019.  You'll have to look at #981 for details concerning the Canon/AF support in exiv2 and and exiftool.